app/views/games/show.html.erb

```diff
@@ -1,102 +1,102 @@
 <head>
 <%= stylesheet_link_tag 'style' %>
- 
+
 <script type="text/javascript">
 var correctCards = 0;
 $( init );
- 
+
 function init() {
   // Hide the success message
   $('#successMessage').hide();
   $('#successMessage').css( {
-    left: '580px',
-    top: '250px',
-    width: 0,
-    height: 0
-  } );
- 
-  // Reset the game
-  correctCards = 0;
-  //$('#cardPile').html( '' );
-  $('#cardSlots').html( '' );
- 
-  // Create the pile of shuffled cards
-  var numbers = [ 1, 2, 3, 4, 5, 6, 7, 8, 9, 10 ];
-  numbers.sort( function() { return Math.random() - .5 } );
- 
-  for ( var i=0; i<<%= @cards.size %>; i++ ) {
-    $("#card"+i).click(function(event) {
+left: '580px',
```

Comment:
This whole file is fucked up.
I've refactored this in my second pull request to put all the javascript in classes in app/assets/javascript/*